modules/govuk/manifests/apps/feedback.pp

```diff
@@ -40,6 +43,7 @@
   $survey_notify_service_api_key = undef,
   $govuk_notify_api_key = undef,
   $govuk_notify_template_id = undef,
+  $govuk_notify_reply_to_id = undef,
```

Comment:
```suggestion
  $govuk_notify_reply_to_id,
```

I wonder if it's better to make this a required argument (and actually perhaps same for the ones above) because then we don't need the if conditional on the environment variable.

Reply:
Sounds sensible 👍